File: 2708.py
# Python 3.4.3

def main():
  qtd_jipes_voltar = 0
  qtd_turistas_voltar = 0

  entrada = input().split()
  movimento = entrada[0]
  if len(entrada) == 2:
    qtd_turistas = int(entrada[1])

  while movimento != 'ABEND':
    if movimento == 'SALIDA':
      qtd_jipes_voltar += 1
      qtd_turistas_voltar += qtd_turistas

    elif movimento == 'VUELTA':
      qtd_jipes_voltar -= 1
      qtd_turistas_voltar -= qtd_turistas

    entrada = input().split()
    movimento = entrada[0]
    if len(entrada) == 2:
      qtd_turistas = int(entrada[1])

  print(qtd_turistas_voltar)
  print(qtd_jipes_voltar)

if __name__ == '__main__':
  main()

File: 2708.c
#include <stdio.h>
#include <string.h>

int main() {
  char m[7];
  int t, qtd_j, qtd_t;
  qtd_j = 0;
  qtd_t = 0;
  
  do {
    scanf("%s", m);
    if(strcmp(m, "ABEND") == 0) {
      break;
    }
    scanf("%d", &t);

    if (strcmp(m, "SALIDA") == 0) {
      qtd_j++;
      qtd_t += t;
    }

    else if(strcmp(m, "VUELTA") == 0) {
      qtd_j--;
      qtd_t -= t;
    }

  } while (strcmp(m, "ABEND") != 0);

  printf("%d\n%d\n", qtd_t, qtd_j);

  return 0;
}